worlds/sc2/Rules.py

```diff
@@ -531,6 +560,7 @@ def the_infinite_cycle_requirement(self, state: CollectionState) -> bool:
             or (
                 self.two_kerrigan_actives(state)
                 and self.basic_kerrigan(state)
+                and self.kerrigan_levels(state, 70)
```

Comment:
This could be lower; maybe 60 or 65 is reasonable (could change the minimum based on if advanced tactics is true). Doesn't have to be updated right now.